app/upload/page.tsx

```
@@ -5,91 +5,373 @@ import { Input } from '@/components/ui/input';
+    setIsUploading(true);
 
     try {
-      const resp = await fetch('/api/patient/upload/before', {
+      // Upload before image
+      const beforeFormData = new FormData();
+      if(!beforeFile || !address)
+      {
+          alert("Missing required data for before image upload.");
+          return; 
+      }
+      beforeFormData.append('image', beforeFile!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)